be/src/exprs/jit/ir_helper.cpp

```diff
@@ -0,0 +1,173 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "exprs/jit/ir_helper.h"
+
+#include "column/vectorized_fwd.h"
+#include "common/status.h"
+#include "common/statusor.h"
+#include "llvm/IR/Constants.h"
+#include "types/logical_type.h"
+
+namespace starrocks {
+
+bool IRHelper::support_jit(const LogicalType& type) {
+    return type == TYPE_BOOLEAN || type == TYPE_TINYINT || type == TYPE_SMALLINT || type == TYPE_INT ||
```

Comment:
why decimal|string|nested type can no be supported?

Reply:
The existing IR for these types of calculations is written through the IRBuilder C++ interface, hence, support for complex types would require a significant amount of work. As for the decimal type, I plan on providing support in the upcoming PR, however, there's currently no scheduled plan for the string and nested types.